main.js

```diff
@@ -91,10 +91,20 @@ var app = {
   }
 }
 
+// added these 2 lines of code below so that I can work on the large card. remove later.
+// app.view = 'details'
+// app.details.item = app.catalog.items[1]
+
+// function 1 - create small cards
 function createCard(item) {
   var $card = document.createElement('div')
   $card.classList.add('card')
 
+  // add attribute to root element of the card
+  var a = document.createAttribute('data-item-id')
```

Comment:
`$element.setAttribute` creates the `AttributeNode` for you, so you don't need to create `a` here.